### DISCUSSION TITLE
Docker support

### DISCUSSION THREAD

**Discussion by rezaei99m** [mermaid-js/mermaid#4029] [Q&A] [answered]
I think it would be a very good idea to add Docker support. Like everything is loaded into the image even live editor itself. I am curious why nobody asks for this feature so far?

**Comment by aloisklink** [comment, marked as answer]
According to the `mermaid-live-editor` README.md, there is already Docker support, see below:

> ## Docker
> 
> ### Run published image
> 
> ```bash
> docker run --publish 8000:80 ghcr.io/mermaid-js/mermaid-live-editor
> ```
>
> _Taken from https://github.com/mermaid-js/mermaid-live-editor#docker_

There is also a [Docker image for the `mermaid-cli` project](https://github.com/mermaid-js/mermaid-cli#use-docker), in case you want to use a program to run Mermaid.
